src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -156,6 +157,7 @@ bool ov::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ov::Model
     REGISTER_DISABLED_PASS(manager, ConvertInterpolate1ToInterpolate4)
 
     auto decomp = manager.register_pass<GraphRewrite>();
+    ADD_MATCHER(decomp, GroupQueryAttentionDecomposition)
```

Comment:
why is it added to main common optimization pass?
As I remember, initially such operation was supposed for NPU only, while CPU/GPU should not support models with GQA in any form.